### PR TITLE
Created User Model and util functions

### LINKED ISSUE #24: BE: User/Auth Model

### PR DESCRIPTION
**Pull request by SilverStrings024**
Created a User model as well as some SQLAlchemy specific utility functions.

### CODE REVIEW

File: server/users/models.py [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+        session.commit()
+
+    # Properties
+
+    @property
+    def full_name(self):
+        return f"{self.first_name} {self.last_name}"
+
+    @property
+    def full_address(self):
```
Comment (dproc96) [inline server/users/models.py]: I like having these methods, but can you update the names to be verbs? Like get_full_address

File: server/users/models.py [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+class User(Base):
+    __tablename__ = "user"
+
+    id = Column(Integer, primary_key=True)
+
+    # General Info
+    first_name = Column(String(50), nullable=False)
+    last_name = Column(String(50), nullable=False)
+    is_chef = Column(Boolean, default=False)
+
```
Comment (dproc96) [inline server/users/models.py]: This will be stored in Profile so you can exclude here

**Comment by SilverStrings024** [comment]
I just noticed I have in incorrect import in users/models.py (Tried to import ManyToMany and it doesn't exist)
Also there is no need for the Session variable on line 27 since fresh sessions are made whenever a session is referenced.

**Review by dproc96 — COMMENTED** [review]
This looks great, if you make the small changes I just mentioned this will be good to merge!

File: server/users/models.py (hunk 2 above)
Comment (SilverStrings024) [reply server/users/models.py]: Okay, thanks!

File: server/users/models.py (hunk 1 above)
Comment (SilverStrings024) [reply server/users/models.py]: Okay, I'll edit it now.

**Pull request merged**